src/GitLink/Extensions/PdbExtensions.cs

```diff
@@ -68,8 +68,15 @@ public static IEnumerable<string> FindMissingOrChangedSourceFiles(this PdbFile p
 
         private static bool AreEqualBuffers(byte[] first, byte[] second)
         {
-            Argument.IsNotNull(nameof(first), first);
-            Argument.IsNotNull(nameof(second), second);
+            if (first == null && second == null)
```

Comment:
And we could store the null result in a variable, even faster, but not sure if such optimizations are really required here